tracer/src/Datadog.Trace.ClrProfiler.Native/dd_profiler_constants.h

```diff
@@ -92,10 +92,21 @@ const WSTRING managed_profiler_full_assembly_version =
 const WSTRING managed_profiler_name = WStr("Datadog.Trace");
 
 const WSTRING nonwindows_nativemethods_type = WStr("Datadog.Trace.ClrProfiler.NativeMethods+NonWindows");
+const WSTRING windows_nativemethods_type = WStr("Datadog.Trace.ClrProfiler.NativeMethods+Windows");
+
 const WSTRING appsec_nonwindows_nativemethods_type = WStr("Datadog.Trace.AppSec.Waf.NativeBindings.NativeLibrary+NonWindows");
+const WSTRING appsec_windows_nativemethods_type = WStr("Datadog.Trace.AppSec.Waf.NativeBindings.NativeLibrary+Windows");
 
 const WSTRING calltarget_modification_action = WStr("CallTargetModification");
 
+#ifdef _WIN32
+const WSTRING native_dll_filename = WStr("DATADOG.TRACE.CLRPROFILER.NATIVE.DLL");
```

Comment:
Can you try to come up with a test to make sure the filename in the source matches the file we actually build?

Reply:
makes sense, will do.